src/api/dimensions.js

```diff
@@ -9,7 +9,7 @@ const selectFromResponse = (response, entity, selectorFn) =>
     typeof selectorFn === 'function' ? selectorFn(response) : response[entity]
 
 // Request functions
-const request = (d2, entity, paramString, { selectorFn } = {}) => {
+const request = ({ d2, entity, paramString }, { selectorFn } = {}) => {
```

Comment:
Changes to `request` and `requestWithPaging` were required by eslint. (max params)